src/main.rs

```diff
@@ -17,13 +17,28 @@ use config::Config;
 
 /// Tiny CLI utility to checkout a recent git branch interactively.
 fn main() -> Result<()> {
-    match run_tui() {
-        Ok(()) => Ok(()),
-        Err(e) => match e.downcast_ref::<SelectBranchError>() {
-            Some(SelectBranchError::Aborted) => process::exit(1),
-            Some(SelectBranchError::Interrupted) => process::exit(2),
-            None => Err(e),
+    let args: Vec<String> = env::args().collect();
+    match args.len() {
+        1 => match run_tui() {
+            Ok(()) => Ok(()),
+            Err(e) => match e.downcast_ref::<SelectBranchError>() {
+                Some(SelectBranchError::Aborted) => process::exit(1),
+                Some(SelectBranchError::Interrupted) => process::exit(2),
+                None => Err(e),
+            },
         },
+        2 => {
+           if ( args[1] == "--version" ) | ( args[1] == "-V" ) {
+               println!("{}", env!("CARGO_PKG_VERSION"));
+               Ok(())
+           } else {
+               Err(Error::msg(format!("Command not recognised {}", args[1])))
```

Comment:
I tend to prefer American english spellings, so for consistency let's go with that here

```suggestion
               Err(Error::msg(format!("Command not recognized {}", args[1])))
```